types/node/fs.d.ts

```diff
@@ -4314,29 +4314,35 @@ declare module "fs" {
      */
     export function cpSync(source: string | URL, destination: string | URL, opts?: CopySyncOptions): void;
 
-    export interface GlobOptions {
+    interface GlobOptionsBase {
```

Comment:
Was there any problem with your original solution, having the derived interfaces inherit from `Omit<GlobOptions, "exclude">` and not needing an additional interface declaration?